jetstream/core/orchestrator.py

```diff
@@ -885,3 +885,17 @@ async def Decode(  # pylint: disable=invalid-overridden-method
         )
         # Reset buffer after flushed.
         buffered_response_list = []
+
+  async def HealthCheck(
+      self,
+      request: jetstream_pb2.HealthCheckRequest,
+      context: Optional[grpc.aio.ServicerContext] = None,
+  ) -> jetstream_pb2.HealthCheckResponse:
+    """HealthCheck."""
+    if context is None:
+      logging.warning(
+          "LLM orchestrator is being used in offline test mode, and will not"
+          " respond to gRPC queries - only direct function calls."
+      )
+    is_live = self._driver.live
```

Comment:
can you share where driver set live status?

Reply:
Yes, it is set here - https://github.com/google/JetStream/blob/e19a7906d8cdf1cae658a4c7c4f6f516aade49f9/jetstream/core/orchestrator.py#L377

Reply:
Thanks for sharing it!